ci-operator/step-registry/assisted/common/gather/assisted-common-gather-commands.sh

```diff
@@ -31,7 +31,7 @@ cat > gather_logs.yaml <<-EOF
         ansible.builtin.shell: |
           source /root/config.sh
           # Get sosreport including sar data
-          sos report --batch --tmp-dir {{ LOGS_DIR }} \
+          sos report --batch --tmp-dir {{ LOGS_DIR }} --all-logs \
```

Comment:
@weliang1 I guess this is mistake update?

Reply:
@zhaozhanqi All `--all-logs`  may be got from rebase. 
Check the latest master version https://github.com/openshift/release/blob/master/ci-operator/step-registry/assisted/baremetal/gather/assisted-baremetal-gather-commands.sh#L45

Reply:
can you fetch latest master code and rebase to remove this changes in this PR?